lib/trino-record-decoder/src/main/java/io/trino/decoder/protobuf/ProtobufUtils.java

```diff
@@ -71,9 +71,15 @@ private ProtobufUtils()
 
```

Comment:
Can we have these preparatory as the first set of commits. then we could reproduce the issue and the next commit will be a fix for that issue ?

Reply:
done, see [Reproduce NPE on Kafka sending when proto file has import](https://github.com/trinodb/trino/pull/15312/commits/5b4932d1031ed8cb39b12500a29ea41640d36784) and [Fix NPE on Kafka sending when proto file has import](https://github.com/trinodb/trino/pull/15312/commits/ec33c828d30198d54c70ce13e3a33e466183f42b)